src/libs/Navigation/AppNavigator/ReportScreenIDSetter.ts

```diff
@@ -45,6 +47,14 @@ function ReportScreenIDSetter({route, reports, policies, navigation, isFirstTime
         // Don't update if there is a reportID in the params already
         if (route?.params?.reportID) {
             App.confirmReadyToOpenApp();
```

Comment:
Is there a reason for this line to be above this logic?

Reply:
Not really, additional lines were removed.